pkgs/top-level/all-packages.nix

```diff
@@ -4797,6 +4797,8 @@ with pkgs;
 
   sov = callPackage ../tools/wayland/sov { };
 
+  sway-new-workspace = callPackage ../tools/wayland/sway-new-workspace { };
+
```

Comment:
```suggestion
```
New top-level packages should go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
Argh, I forgot about the RFC 140. Thanks!